.github/workflows/rust.yml

```diff
@@ -0,0 +1,94 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - '**'
+
+
+env:
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: 1
+
+jobs:
+  style:
+    name: Check Style
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+          components: rustfmt, clippy
```

Comment:
I'd skip clippy. I don't personally find it useful, triggering more false positives than catching problems.

Reply:
There are a bunch of things that clippy is able to catch and highlight syntax that one might want to avoid because it is ambiguous. I was aided a lot by clippy when i was learning rust, and I think that it helps a lot more than it hinders. The rust compiler and clippy taught me a lot about how to write rust code in a good way when I was beginning my rust journey.

This being a open source project in the top 200 all-time downloaded crates, I think it's important to be helpful to any PR handling. Having instant feedback from build-checks to new rustaceans is a super welcoming feeling in my opinion. 

There are no clippy warnings in this project, so no changes to the code are being suggested now. Do you still think that clippy is bad for this project?